src/test/kotlin/org/jetbrains/bio/big/BedEntryTest.kt

```diff
@@ -132,8 +132,35 @@ class BedEntryTest {
 
     @Test fun unpackBed3as12() {
         val bedEntry = BedEntry("chr1", 1, 100, "")
+        try {
+            bedEntry.unpack(fieldsNumber = 12)
+        } catch (e: BedEntryUnpackException) {
+            assertEquals(3, e.fieldIdx, "Missing field index is wrong")
+            return
+        }
+        assertTrue(false, "bed3 parsed as bed12 without throwing")
```

Comment:
Please use `@Test(expected = BedEntryUnpackException::class)` or 
```
class FooTest {
@Rule @JvmField var thrown = ExpectedException.none()

 @Test fun testSmthFails() {
        smth.init()
        thrown.expect(IllegalArgumentException::class.java)
        thrown.expectMessage("Smth went wrong at 3-rd arg")
        smth.doDangerThing()
    }
}
```

Reply:
Thanks for the suggestion! Done.